src/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <!-- <link rel="stylesheet" href="./style.css"> -->
+    <title>Todo List Project</title>
+</head>
+<body>
+    <main class="main">
+        <section class="big-cont">
+            <div class="title-cont">
+                <h3 class="title">Today's To Do</h3>
+                <i class="bi bi-arrow-clockwise"></i>
+            </div>
+            <div class="add-cont">
+                <form id="formSection">
+                    <input type="text" id="newTodo" name="addTo" placeholder="Add to your list...">
+                   <button type="submit"> <i class="bi bi-arrow-return-left"></i></button>
+                </form>
+                
+            </div>
+            <div id="todo-list">
+                <!-- <div class="todo"> -->
+                    <!-- <i class="bi bi-circle"></i> -->
+                    <!-- <i class="bi bi-check-lg"></i> -->
+                    <!-- <p class=""></p> -->
+                    <!-- <i class="bi bi-three-dots-vertical"></i> -->
+                    <!-- <i class="bi bi-trash"></i> -->
+                <!-- </div> -->
```

Comment:
## [Optional]

Kindly please do not leave unused code as comments in your code base, unused code as comments make your codes not clean.